taskcluster/tc-tests-utils.sh

```diff
@@ -811,7 +811,7 @@ do_bazel_build()
   fi;
 
   bazel ${BAZEL_OUTPUT_USER_ROOT} build \
-    -s --explain bazel_monolithic.log --verbose_explanations --experimental_strict_action_env --config=monolithic -c opt ${BAZEL_BUILD_FLAGS} ${BAZEL_TARGETS}
+    -s --explain bazel_monolithic.log --verbose_explanations --experimental_strict_action_env --workspace_status_command="bash native_client/bazel_workspace_status_cmd.sh" --config=monolithic -c opt ${BAZEL_BUILD_FLAGS} ${BAZEL_TARGETS}
```

Comment:
So, no spurious rebuilds with that ?

Reply:
Nope. All the spurious rebuilds tests that were failing before are now green.